docs/content/latest/develop/best-practices.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Best practices
```

Comment:
Some more things we could have in this section:
- resource constrained environments and which flags we can tune for those
- for far away geos, should recommend tuning some raft flags (number of missed heartbeats and leader lease duration)

Reply:
> resource constrained environments and which flags we can tune for those
Those are in https://docs.yugabyte.com/latest/deploy/checklist/ (comment if we need more)

> for far away geos, should recommend tuning some raft flags (number of missed heartbeats and leader lease duration)

I will try something out. But can be in v2.